.github/workflows/release.yml

```diff
@@ -79,12 +84,21 @@ jobs:
       - 
         name: Build Image without automation library version tag
         uses: docker/build-push-action@v2
-        if: ${{ !contains(matrix.image-name, 'puppeteer') && !contains(matrix.image-name, 'playwright') || env.RELEASE_TAG !='latest'}}
+        if: ${{ matrix.image-name != 'node' && env.RELEASE_TAG !='latest'}}
         with:
           context: ./${{ matrix.image-name }}
           file: ./${{ matrix.image-name }}/Dockerfile
           load: true
           tags: apify/actor-${{ matrix.image-name }}:${{ env.RELEASE_TAG }}
+      - 
+        name: Build image with node version.
+        uses: docker/build-push-action@v2
+        if: ${{ matrix.image-name == 'node' }}
+        with:
+          context: ./${{ matrix.image-name }}
+          file: ./${{ matrix.image-name }}/Dockerfile
+          load: true
+          tags: apify/actor-${{ matrix.image-name }}:${{ env.RELEASE_TAG }},apify/actor-${{ matrix.image-name }}:${{ env.NODE_VERSION }}
```

Comment:
I guess we would overwrite the `:14` images on a beta release like that, no? Because we would release `:beta`, that's fine, and then `:14`, which would overwrite the previous 14, even if it were latest.

Reply:
Good point. That would definitely happen I will fix it with the refactor.